apisix/plugins/fault-injection.lua

```diff
@@ -64,6 +119,25 @@ local function sample_hit(percentage)
 end
 
 
+local function vars_match(vars, ctx)
+    local match_result
+    for _, var in ipairs(vars) do
+        local expr, err = expr.new(var)
+        if err then
+            core.log.error("vars expression does not match: ", err)
```

Comment:
Should be `failed to create vars expression`, we haven't matched it yet.

Reply:
updated.